pysatNASA/tests/test_instruments.py

```diff
@@ -26,3 +35,56 @@ class TestInstruments(InstLibTests):
     instrument test class.
 
     """
+
+    @pytest.mark.second
+    # Need to maintain download mark for backwards compatibility.
+    # Can remove once pysat 3.1.0 is released and libraries are updated.
+    @pytest.mark.load_options
+    @pytest.mark.download
```

Comment:
```suggestion
```
We can drop these since pysat 3.0.2 is out and a requirement.  Verified in pysatCDAAC tests that these are not needed.  The new initialize statement is run before these are defined, so we don't need to repeat info here.